plugins/knife/knife-box-create.rb

```diff
@@ -80,6 +82,12 @@ class BoxCreate < Chef::Knife
       :default => false,
       :description => "Does this deployment have a Ralf?"
 
+    option :standalone,
+      :long => "--standalone",
+      :boolean => true,
+      :default => false,
+      :description => "Should this box have etcd configuration applied?"
```

Comment:
Is this the right question? I'd expect "Have you created a deployment with knife deployment resize?" or similar (though obviously then true/false are reversed).